unittests/kv_tests.cpp

```diff
@@ -646,27 +643,17 @@ class kv_tester : public tester {
    }
 
    void test_max_iterators() {
-      BOOST_TEST_REQUIRE(set_kv_limits(N(eosio.kvram), 1024, 1024, 5) == "");
       BOOST_TEST_REQUIRE(set_kv_limits(N(eosio.kvdisk), 1024, 1024, 7) == "");
       // individual limits
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 5, false}}) == "");
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 6, false}}) == "Too many iterators");
       BOOST_TEST(iterlimit({{N(eosio.kvdisk), 7, false}}) == "");
       BOOST_TEST(iterlimit({{N(eosio.kvdisk), 2000, false}}) == "Too many iterators");
-      // both limits together
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 5, false}, {N(eosio.kvdisk), 7, false}}) == "");
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 6, false}, {N(eosio.kvdisk), 7, false}}) == "Too many iterators");
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 5, false}, {N(eosio.kvdisk), 8, false}}) == "Too many iterators");
       // erase iterators and create more
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 4, false}, {N(eosio.kvdisk), 6, false},
-                            {N(eosio.kvram), 2, true}, {N(eosio.kvdisk), 2, true},
-                            {N(eosio.kvram), 3, false}, {N(eosio.kvdisk), 3, false}}) == "");
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 4, false}, {N(eosio.kvdisk), 6, false},
-                            {N(eosio.kvram), 2, true}, {N(eosio.kvdisk), 2, true},
-                            {N(eosio.kvram), 3, false}, {N(eosio.kvdisk), 4, false}}) == "Too many iterators");
-      BOOST_TEST(iterlimit({{N(eosio.kvram), 4, false}, {N(eosio.kvdisk), 6, false},
-                            {N(eosio.kvram), 2, true}, {N(eosio.kvdisk), 2, true},
-                            {N(eosio.kvram), 4, false}, {N(eosio.kvdisk), 3, false}}) == "Too many iterators");
+      BOOST_TEST(iterlimit({{N(eosio.kvdisk), 6, false},
```

Comment:
All references to kvdisk will be removed at the final phase of removing DISK mode. It depends on removing 
database ID in the test contract. Jeff will do that.

Reply:
So why did we remove eosio.kvram and not eosio.kvdisk?

Reply:
We could remove eosio.kvdisk and keep eosio.kvram for now
But since in original design, only kvdisk would trigger RocksDB.
That's why I kept kvdisk for easier testing. It will be removed once contract and CDT are done.